File: Phyton/Vetor/aprovados.py
n = int(input("Quantos alunos serão digitados ?"))

nome = [0 for x in range(n)]
nota1 = [0 for x in range(n)]
nota2 = [0 for x in range(n)]

for i in range(n):
    print(f"Digite o nome, primeira nota e segunda nota do {i}º aluno:")
    nome[i] = input()
    nota1[i] = float(input())
    nota2[i] = float(input())

print("Alunos aprovados: ")
for i in range(n):
   if (nota1[i]+nota2[i])/2 >= 6.0:
        print(nome[i])

File: C/Vetor/Aprovados.c
#include <stdio.h>
#include <string.h>
void limpar_entrada() {
 char c;
 while ((c = getchar()) != '\n' && c != EOF) {}
}
int main(){
int i,n;
    printf("Quantos alunos serao digitados? ");
    scanf("%d", &n);
        double Nota1[n], Nota2[n], Media[n];
        char Nome[n][50];

                    for(i=0; i<n; i++){
                        printf("Digite nome, primeira e segunda nota do %do aluno:\n", i+1);
                        limpar_entrada();
                        gets(Nome[i]);
                        scanf("%lf", &Nota1[i]);
                        scanf("%lf", &Nota2[i]);
                        Media[i] = (Nota1[i] + Nota2[i])/2;
                        }

    printf("ALUNOS APROVADOS: \n");
    for(i=0;i<n;i++){
        if(Media[i]>=6.0){
            printf("%s\n", Nome[i]);
        }
    }
}
